src/utils/sendWork.js

```diff
@@ -19,13 +19,19 @@ const sendWork = async (experimentId, timeout, body, requestProps = {}) => {
 
   const timeoutDate = moment().add(adjustedTimeout, 's').toISOString();
 
+  let authJWT = null;
+  try {
+    const currentSession = await Auth.currentSession();
+    authJWT = currentSession.getIdToken().getJwtToken();
+  } catch (e) {
+    authJWT = null;
+  }
```

Comment:
that looks like the same code as the one we have in `fetchApi`, is it worth extracting it as a common component?

Reply:
Yeah, good point. I refactored it.